bentoml/_internal/utils/cattr.py

```diff
@@ -26,23 +27,100 @@ def datetime_structure_hook(dt_like: str | datetime, _: t.Any) -> datetime:
         raise Exception(f"Unable to parse datetime from '{dt_like}'")
 
 
-bentoml_cattr.register_structure_hook_factory(
-    attr.has,
-    lambda cls: make_dict_structure_fn(
-        cls,
-        bentoml_cattr,
-        _cattrs_forbid_extra_keys=getattr(cls, "__forbid_extra_keys__", False),
-    ),
-)
-
-bentoml_cattr.register_unstructure_hook_factory(
-    attr.has,
-    lambda cls: make_dict_unstructure_fn(
-        cls,
-        bentoml_cattr,
-        _cattrs_omit_if_default=getattr(cls, "__omit_if_default__", False),
-    ),
-)
-
-bentoml_cattr.register_structure_hook(datetime, datetime_structure_hook)
-bentoml_cattr.register_unstructure_hook(datetime, lambda dt: dt.isoformat())
+def register_global_structure_hook() -> None:
```

Comment:
let's don't do this change. it's better to keep the structure/unstructure hook contextual to where the class is defined